src/Runner.Sdk/Util/IOUtil.cs

```diff
@@ -46,6 +46,21 @@ public static T LoadObject<T>(string path)
             return StringUtil.ConvertFromJson<T>(json);
         }
 
+        public static T LoadObject<T>(string path, bool required)
+        {
+            string json = File.ReadAllText(path, Encoding.UTF8);
+            if (string.IsNullOrEmpty(json))
+            {
+                throw new InvalidOperationException($"File {path} is empty");
```

Comment:
@TingluoHuang any suggestions for exception that we should throw :relaxed: ?

Reply:
do we want to merge `public static T LoadObject<T>(string path, bool required)` merge with `public static T LoadObject<T>(string path)` to be something like `public static T LoadObject<T>(string path, bool required = false)`?

Reply:
`ArgumentNullException` or `ArgumentException` both seems fine to me.

Reply:
Applied `ArgumentNullException` if file is empty or null, `ArgumentException` if convert from json returns null